src/renderers/dom/shared/CSSProperty.js

```diff
@@ -33,12 +33,14 @@ var isUnitlessNumber = {
   widows: true,
   zIndex: true,
   zoom: true,
+  animationIterationCount: true,
 
   // SVG-related properties
   fillOpacity: true,
   strokeDashoffset: true,
   strokeOpacity: true,
   strokeWidth: true,
+
```

Comment:
Thanks, now can you get rid of this extraneous line and squash to a single commit?